lib/cashbox/model/payment_method.rb

```
@@ -28,5 +28,13 @@ def direct_debit?
     def last_digits
       self[type.underscore].last_digits
     end
+
+    def card_network
+      return "not a card" unless type == "CreditCard"
```

should throw an error for this